daal4py/sklearn/cluster/_k_means_0_23.py

```diff
@@ -174,6 +174,14 @@ def _daal4py_k_means_fit(X, nClusters, numIterations,
     if numIterations < 0:
         raise ValueError("Wrong iterations number")
 
+    def is_string(s, target_str):
+        return isinstance(s, str) and s == target_str
```

Comment:
Is there a reason why `isinstance` check needed?

Reply:
"n_init" accepts int and 'auto'. "cluster_centers_0" (which is used as an alias of "init" in scikit-learn) can accept str and array-like. The check may be avoided for n_init, but I think "cluster_centers_0" needs the check.